Makefile

```diff
@@ -1,13 +1,13 @@
 COMMIT=$(shell git rev-parse --short HEAD)$(shell [[ $$(git status --porcelain --ignored) = "" ]] && echo -clean || echo -dirty)
 
 # all is the default target to build everything
-all: clean build azure-controllers etcdbackup sync
+all: clean build azure-controllers etcdbackup sync metricsbridge
```

Comment:
Can't see the new target anywhere in the Makefile

Reply:
Hrm, seems to be there it's just your master branch is old.

/lgtm

Reply:
github cache 
https://github.com/openshift/openshift-azure/blob/master/Makefile#L44